help/export-your-organization.md

```diff
@@ -62,37 +71,78 @@ with lots of messages or uploaded files.
 subscriptions exported, but will otherwise be treated as new users after
 import.
 
+{!not-human-export-format.md!}
+
 ## Full export without member consent
 
 {!owner-only.md!}
 
-This export is limited to paid Zulip Cloud Standard customers, though in rare
-cases may be available to other organizations in case of due legal process.
+This export is limited to paid [Zulip Cloud Standard][standard]
+customers, though in rare cases may be available to other
+organizations in case of due legal process.
 
-To start this export, email support@zulip.com with your zulip.com
+To start this export, email
+[support@zulip.com](mailto:support@zulip.com) with your zulipchat.com
 URL, asking for a full export without member consent.
 
-You'll also need to email us evidence that you have authority to read
-members' direct messages. Typically, this will be because the zulip.com
-URL is administered by a corporation, and you are an official
-representative of that corporation. By requesting and approving this export,
-you will also assume full legal responsibility that the appropriate employment
-agreements and corporate policy for this type of export are in place. Note
-that many countries have laws that require employers to notify employees of
-their use of such an export.
+You'll also need to provide evidence that you have authority to read
+members' direct messages. Typically, this will be because the
+zulip.com URL is administered by a corporation, and you are an
+official representative of that corporation. By requesting and
+approving this export, you will also assume full legal responsibility
+that the appropriate employment agreements and corporate policy for
+this type of export are in place. Note that many countries have laws
+that require employers to notify employees of their use of such an
+export.
+
+{!not-human-export-format.md!}
+
+## Compliance export
+
+{!owner-only.md!}
+
+This export is limited to paid [Zulip Cloud Standard][standard]
+customers, though in rare cases may be available to other
+organizations in case of due legal process. As with full exports
+without member consent, you will also need to provide evidence that
+you have authority to read members' direct messages.
+
+To start this export, email
+[support@zulip.com](mailto:support@zulip.com), from the email address
+you use to log into Zulip.  Please specify:
+
+1. That you're asking for a compliance export.
+
+1. The zulipchat.com URL for your organization
+
+1. What limits you would like on the export.  Currently, compliance
+   exports can apply any combination of the below filters:
+
+    - Message sender
+    - Message recipient
+    - Message contents, by specific keywords
+    - Sent timestamp before, after, or between dates
+
+    If you need other limits, please ask.
+
+1. What output format you would prefer: CSV or JSON.
+
+1. If you would like a copy of all attachments referenced in the
+   matching messages.
```

Comment:
Should we link to the documentation for doing this for self-hosters here?

Reply:
I feel like they've already been diverted, given the first lines of the document:

```

!!! warn ""
    These instructions are specific to the hosted Zulip Cloud service.
    If you're running your own server, you may be looking for our
    documentation on [server export and import][export-and-import] or
    [server backups][production-backups].

Reply:
Hmm, yeah; I suppose we could add a "or server compliance exports" there if we think that'll be a common thing folks are looking for. 

Speaking of which, do we want a link back to this zulip.com/help/ page from the self-hosted compliance section, just in case someone lands there from google?

Reply:
Neither of those feels like a high likelihood thing, to me.  For the former, I think we're generally categorizing compliance exports as "server exports" already, so the existing link covers it.  For the latter, there are a _lot_ of places we could conceivably link to `/help/` from the self-hosting side "just in case" and compliance exports don't seem high on that list to me.

Reply:
@alya thoughts? I'm mostly thinking that both of these pages are possible places to land after googling "compliance exports" or clicking a link that we will probably eventually add to our /for/business/ pages about this feature.